coap-core/src/test/java/com/mbed/coap/transport/udp/DatagramSocketTransportTest.java

```diff
@@ -66,61 +65,48 @@ public void initializingWithStateException() throws IOException {
 
             trans.start(mock(CoapReceiver.class));
 
-            try {
-                trans.setReuseAddress(true);
-                fail();
-            } catch (Exception e) {
-                assertTrue(e instanceof IllegalStateException);
-            }
-            try {
-                trans.setSocketBufferSize(1234);
-                fail();
-            } catch (Exception e) {
-                assertTrue(e instanceof IllegalStateException);
-            }
         } finally {
             trans.stop();
         }
     }
 
     @Test
     public void initializeWithParameters() throws Exception {
```

Comment:
I think this test does not make sense any more, just remove it.

Reply:
I think it still tests the socket buffer size. We need it for coverage.

Reply:
removed.